app/controllers/users_controller.rb

```diff
@@ -2,24 +2,28 @@ class UsersController < ApplicationController
   include Accessible
 
   def new
+    @person = Person.new
+    @person.personable = User.new
```

Comment:
One small simplification is that you can just use one of these lines and then push the initialization of the association down into the model. If we definitely want `@person` as the end result, then within Person.rb

```
after_initialize :set_default_personable

private

def set_default_personable
  self.personable ||= User.new
end
```